### PR TITLE
docker 1.6 Labels support

### PR DESCRIPTION
**Pull request by cflynn07**

### CODE REVIEW

**Comment by cflynn07** [comment]
https://docs.docker.com/reference/api/docker_remote_api_v1.18/#inspect-a-container
Are we sure we're responding with the correct JSON structure for container-inspect mocking? I ask because I see the documentation provide many of the properties we're returning under a "Config" parent property and we don't appear to be replicating that.

**Comment by bkendall** [comment]
remove `.jshintrc`. this uses `eslint`, it's in the `package.json` and you can run it w/ `npm run lint`

File: lib/middleware.js [hunk 1]
```diff
@@ -161,6 +166,7 @@ var middlewares = module.exports = {
       'Cmd': [],
       'Dns': null,
       'Image': null,
+      'Labels': keypath.get(req, 'body.Labels') || {},
```
Comment (bkendall) [inline lib/middleware.js]: below, there's a `data = assign(data, req.body);`. You can default this `'Lables'` to `{}` if you like, and let the `assign` set the value from the `body`

Comment (cflynn07) [inline lib/middleware.js]: When creating a container, "Labels" is a root level key on the body -- but when inspecting a container docker returns "Labels" under "Config" -- making labels a 2nd level child of the response

**Pull request merged**